<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>team_before</class>
 <widget class="QWidget" name="team_before">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1166</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#team_before
{
background-color:black;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label
{
color:white;
}</string>
   </property>
   <property name="text">
    <string>What do you want to do with the team?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_admin_to_team_pbn">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>130</y>
     <width>211</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#add_admin_to_team_pbn{
background-color:white;
border-radius:10px;
font-size:20px;
}

#add_admin_to_team_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Adding admin to team</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_admin_pbn">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>260</y>
     <width>211</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#delete_admin_pbn{
background-color:white;
border-radius:10px;
font-size:25px;
}

#delete_admin_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Deleting admin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_admins_pbn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>211</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#view_admins_pbn{
background-color:white;
border-radius:10px;
font-size:25px;
}

#view_admins_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Viewing admins</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_team_task_pbn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>211</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#change_team_task_pbn{
background-color:white;
border-radius:10px;
font-size:20px;
}

#change_team_task_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Changing task of team</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_task_pbn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>211</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#view_task_pbn{
background-color:white;
border-radius:10px;
font-size:25px;
}

#view_task_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Viewing tasks</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_member_pbn">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>540</y>
     <width>211</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#add_member_pbn{
background-color:white;
border-radius:10px;
font-size:25px;
}

#add_member_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Adding member</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_member_pbn">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>400</y>
     <width>211</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#delete_member_pbn{
background-color:white;
border-radius:10px;
font-size:25px;
}

#delete_member_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Deleting member</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_member_pbn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>460</y>
     <width>211</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#view_member_pbn{
background-color:white;
border-radius:10px;
font-size:25px;
}

#view_member_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Viewing members</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_member_task_pbn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>211</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#change_member_task_pbn{
background-color:white;
border-radius:10px;
font-size:18px;
}

#change_member_task_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Changing task of member</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_member_task_pbn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>211</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#view_member_task_pbn{
background-color:white;
border-radius:10px;
font-size:19px;
}

#view_member_task_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Viewing task of member</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_add_mem">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>510</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_del_mem">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>370</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_del_adm">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>230</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_add_adm">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>100</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_view_taskmem">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_chan_taskmem">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_chan_taskteam">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_view_taskteam">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="ted_view_mem">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>346</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="ted_view_adm">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>120</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>250</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
